Common/cpp/Tools/JSScheduler.h

```diff
@@ -37,6 +37,7 @@ class JSScheduler {
 
  public:
   const std::function<void(Job)> scheduleOnJS = nullptr;
+  const std::shared_ptr<CallInvoker> getJSCallInvoker() const;
```

Comment:
Let's move this declaration right after `JSScheduler` constructor (we already have `public:` there).